ui/mirage/config.js

```diff
@@ -887,7 +891,12 @@ export default function () {
 
   //#region Variables
 
-  this.get('/vars', function (schema, { queryParams: { namespace } }) {
+  this.get('/vars', function (schema, { queryParams: { namespace, prefix } }) {
+    if (prefix === 'nomad/job-templates') {
+      return schema.variables
+        .all()
+        .filter((v) => v.path.includes('nomad/job-templates'));
+    }
```

Comment:
`question`: Do we have a branching issue, again? I think I already addressed this in a [previous PR](https://github.com/hashicorp/nomad/pull/15827/commits/d2ebb99c0b7a519fee2cd92c048de777eb13f756#).

Reply:
Yep, but that is not yet merged to epic/job-templates which this is based from. This duplicates that code and we should not have any merge conflicts.